test/util/util.go

```diff
@@ -0,0 +1,346 @@
+/*
+Copyright 2023 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
```

Comment:
Many of the funcs in this file are not general utilities, they are only for use in gingko tests. Maybe have a `testutil` package? If not, at least name this file something more specific. "util/util" says nothing.

Reply:
It's not util/util.  It's test/util/util.go.  Or test/util when imported.  It is located under test and is included as "github.com/kubestellar/kubestellar/test/util".  Isn't changing util to testutil redundant?  You'll end up with "github.com/kubestellar/kubestellar/test/testutil".  If you think this is important, I'll make the change.

Reply:
Sorry, I overlooked the "test" part.